src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -34,154 +37,249 @@
   private String userStaff = credentials.userNameStaff;
   private String userStaffPassword = credentials.passwordStaff;
 
-  private ChatPageObject openChatForUser(String userName, String password) {
+  private final String currentBrowserTab() {
+    return driver.getWindowHandle();
+  }
+
+  private ChatPage openChatForUser(String userName, String password) {
     WikiBasePageObject base = new WikiBasePageObject();
     base.loginAs(userName, password, wikiURL);
-    return base.openChat(wikiURL);
+    return new ChatPage().open();
   }
 
-  @DontRun(env = {"preview", "dev", "sandbox"})
-  @Test(groups = {"Chat_001", "Chat", "ChatTests"})
-  @RelatedIssue(issueID = "MAIN-6071")
-  public void Chat_001_twoUserEnterChat() {
-    ChatPageObject chatUserOne = openChatForUser(userOne, userOnePassword);
-    chatUserOne.verifyChatPage();
+  @DontRun(env = {"preview", "dev"})
+  @Test(groups = "ChatTests")
+  public void dropDownMenuForRegularUser() {
+    ChatPage chatUserOne = openChatForUser(userOne, userOnePassword);
 
     switchToWindow(1);
     new SpecialVersionPage().open();
-    ChatPageObject chatUserTwo = openChatForUser(userTwo, userTwoPassword);
-    chatUserTwo.verifyChatPage();
-    switchToWindow(0);
+    openChatForUser(userTwo, userTwoPassword);
 
-    chatUserOne.verifyUserJoinToChatMessage(userTwo);
+    switchToWindow(0);
+    chatUserOne.clickOnDifferentUser(userTwo);
+    Assertion.assertTrue(chatUserOne.isRegularUserDropdownDisplayed(), "REGULAR USER DROBDOWN IS NOT DISPLAYED");
   }
 
-  @DontRun(env = {"preview", "dev", "sandbox"})
-  @Test(groups = {"Chat_002", "Chat", "ChatTests"})
-  public void Chat_002_dropDownMenuForRegularUser() {
-    ChatPageObject chatUserOne = openChatForUser(userOne, userOnePassword);
-    chatUserOne.verifyChatPage();
+  @DontRun(env = {"preview", "dev"})
+  @Test(groups = "ChatTests")
+  public void verifySwitchingBetweenMainAndPrivateSections() {
+    ChatPage chatUserOne = openChatForUser(userOne, userOnePassword);
 
     switchToWindow(1);
     new SpecialVersionPage().open();
-    openChatForUser(userTwo, userTwoPassword);
+    ChatPage chatUserTwo = openChatForUser(userTwo, userTwoPassword);
+    chatUserTwo.writeOnChat(MESSAGE_ON_MAIN_CHAT);
 
     switchToWindow(0);
-    chatUserOne.verifyNormalUserDropdown(userTwo);
+    Assertion.assertTrue(chatUserOne.isMessageOnChat(MESSAGE_ON_MAIN_CHAT), "MESAGE ON CHAT IS NOT DISPLAYED");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "MESAGE ON CHAT IS NOT DISPLAYED" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)